bigbone/src/main/kotlin/social/bigbone/api/entity/data/Privacy.kt

```diff
@@ -0,0 +1,11 @@
+package social.bigbone.api.entity.data
+
+/**
+ * Specifies the privacy or visibility for new posts.
+ */
+enum class Privacy(val value: String) {
```

Comment:
We already maintain a `Visibility` type in `bigbone/src/main/kotlin/social/bigbone/api/entity/Status.kt`. I'd suggest that you extract that `Visibility` enum into the `bigbone/src/main/kotlin/social/bigbone/api/entity/data/Visibility.kt` package and remove this enum afterwards.